app/config/services/community.xml

```diff
@@ -77,9 +77,7 @@
         </service>
 
         <!-- Procuration -->
-        <service id="app.procuration.request_flow" class="AppBundle\Procuration\ProcurationRequestFlow">
-            <argument type="service" id="session" />
-        </service>
+        <service autowire="true" id="AppBundle\Procuration\ProcurationRequestSession" />
```

Comment:
You can add this declaration to `services.xml` if you want. 
I created this file to be our autowire/autoconfig that is shipped with SF by default now.
The intention is to migrate namespace by namespace as we developed.

Reply:
Didn't no about that 👌, done. Thanks!